proc-macros/tests/ui/incorrect/rpc/rpc_deprecated_method.stderr

```diff
@@ -1,11 +1,11 @@
-error: use of deprecated method `DeprecatedClient::async_method`: please use `new_method` instead
-  --> $DIR/rpc_deprecated_method.rs:63:20
+error: use of deprecated associated function `DeprecatedClient::async_method`: please use `new_method` instead
```

Comment:
Which rustc version are you using to get these? 

This repo is using rustc stable and I guess you are using nightly or something?

Reply:
I was getting some errors during the build because of the rustc version, and I wrongly used the TRYBUILD=overwrite flag 😅, now it's fixed.